Doc/whatsnew/3.11.rst

```diff
@@ -123,6 +123,12 @@ as well as complex arithmetic expressions:
 See :pep:`657` for more details. (Contributed by Pablo Galindo, Batuhan Taskaya
 and Ammar Askar in :issue:`43950`.)
 
+.. note::
+
+   This feature requires storing column positions in code objects which may
+   increase disk usage of compiled Python files or interpreter memory usage.
+   It can be disabled with the ``-X no_debug_ranges`` command line flag.
```

Comment:
```suggestion
   This feature requires storing column positions in code objects which may
   result in a small increase of disk usage of compiled Python files or interpreter memory usage.
   To avoid storing the extra information or to deactivate printing the extra traceback information,
   the feature can be fully disabled with the ``-X no_debug_ranges`` command line flag or the ``PYTHONNODEBUGRANGES`` environment variable.
```

Reply:
Copy this also to the note in the `codeobject.co_positions()` docs

Reply:
Done.